<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.017, x2:0, y2:0.018, stop:0 rgba(15, 176, 7, 255), stop:1 rgba(238, 238, 238, 255));
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>+  NUEVO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>EDITAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="placeholderText">
     <string>  Filtrar...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 31, 2);
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBuscar">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(162, 162, 162);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>BUSCCAR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>661</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMostrar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(38, 70, 255);
border-radius:10px;
font-size:20px;
color:white;
font-family:&quot;Arial Narrow&quot;;</string>
    </property>
    <property name="text">
     <string>MOSTRAR PRODUCTOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color: rgb(157, 157, 157)</string>
    </property>
    <property name="text">
     <string>ION SAC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNew_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
color: rgb(255, 255, 255);
font-size: 20px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNew_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
color: rgb(255, 255, 255);
font-size: 20px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Ayuda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNew_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);
color: rgb(255, 255, 255);
font-size: 20px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Web Site</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>276</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
color:rgb(127, 127, 127);</string>
    </property>
    <property name="text">
     <string>Modo Oscuro</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
